db/disk/submission.go

```diff
@@ -214,7 +214,7 @@ func (this *backend) GetRecentSubmissionSurvey(assignment *model.Assignment, fil
 
 func (this *backend) GetSubmissionContents(assignment *model.Assignment, email string, shortSubmissionID string) (*model.GradingResult, error) {
     var err error;
-
+    
```

Comment:
(Unresolving comment.)

This has not been resolved.
Be really careful to miss something that you say is resolved (reviews hate that).

Reply:
Sorry, I mistook stray spacing for an extra line. Will not happen again.

Reply:
np.
Make sure to read PR comments very matter-of-factly (all denotation, no connotation),
as they are generally written as just stating facts and not emotionally.
This applies to pretty much all PRs, not just mine.

(I just want to make sure, since it can be easy to accidentally read them and pull out emotion that is not there. And that can make PRs really traumatizing.)

Also, now you are in a peculiar spot where you commented on a thread, but also need to resolve this thread
(because the review will have nothing to do with it).
In general you want to avoid comments like this, and just resolve it.
If you really need to apologize for something (which is rare in code), it's better to do it in person, via some other communication, or in the general comments for the PR (on the main PR page).

If you do need to make small comments like this, then make the comment but still resolve it so the reviewer doesn't see it as an action item.
(Many reviewers will still look at all the resolved comments from last round to page in the PR.)